css/selectors.json

```diff
@@ -116,6 +116,15 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:checked"
   },
+  ":blank": {
+    "syntax": ":blank",
+    "groups": [
+      "Pseudo-classes",
+      "Selectors"
+    ],
+    "status": "standard",
```

Comment:
This is currently `experimental`, as no major browser is currently shipping this unprefixed in a stable release.

(run `git checkout -B css-selectors-blank 8c19a272b13a058e96e83114d9e75f53ac57c29e` and `git push -f` to roll&#x2011;back https://github.com/mdn/data/commit/b4bd83d4c80678f2bd8d3d2641fbe4ec11f2f104)